backend/modules/instruments/services.py

```
@@ -0,0 +1,97 @@
+from typing import TypedDict
+
+from modules.instruments.repositories import YfinanceRepository
+from modules.instruments.schemas import (
+    InstrumentBasicInfoSchema,
+    InstrumentDetailedInfoSchema,
+)
+from modules.prices.constants import YFinanceTimeInterval
```

Ruff (0.11.9)

`modules.prices.constants.YFinanceTimeInterval` imported but unused

Remove unused import: `modules.prices.constants.YFinanceTimeInterval`

(F401)